gradle/dependencies.gradle

```diff
@@ -13,7 +13,7 @@ ext {
   // version which we should use in this build
   def mapboxNavigatorVersion = System.getenv("FORCE_MAPBOX_NAVIGATION_NATIVE_VERSION")
   if (mapboxNavigatorVersion == null || mapboxNavigatorVersion == '') {
-      mapboxNavigatorVersion = '132.0.0'
+      mapboxNavigatorVersion = '132.2.0'
```

Comment:
Unit test compilation is going to fail. See https://github.com/mapbox/mapbox-navigation-android/pull/7167 for the fix.